assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -94,7 +98,10 @@ function DisplayElement({ element }) {
           opacity: opacity ? opacity / 100 : null,
         }}
       >
-        <Display element={replacementElement || element} box={box} />
+        <Display element={element} box={box} />
```

Comment:
Is there any reason to display `Display` if there's a replacement element?

Reply:
I have the same question.

Reply:
So that there's something behind the new image as it loads, otherwise we get the background flash